### PR TITLE
Add a debian package for the GDK plugin

### PR DESCRIPTION
**Pull request by veluca93**

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -19,6 +20,7 @@ Build-Depends:
  libpng-dev,
  libwebp-dev,
  pkg-config,
+ xdg-utils,
```
Comment (deymo) [inline debian/control:23]: This is for the test only, right? I think we should install xvfb for this test to pass in the builders. Did you test this in github?

Comment (deymo) [reply debian/control:23]: I'm also ok not running this test in the Debian package build.

Comment (veluca93) [reply debian/control:23]: No no, this is for xdg-mime, which apparently gets executed I think in cmake install... The pipeline fails without this either way (and works with this line ;))

File: debian/libjxl-gdk-pixbuf.install [hunk 2]
```diff
@@ -0,0 +1,3 @@
+usr/lib/x86_64-linux-gnu/*/*/loaders/*
```
Comment (deymo) [inline debian/libjxl-gdk-pixbuf.install:1]: This wouldn't work for other archs, right?

Comment (veluca93) [reply debian/libjxl-gdk-pixbuf.install]: Very true. Fixed.

**Review by deymo — APPROVED** [review]

**Pull request merged**